apps/src/music/blockly/MusicBlocklyWorkspace.js

```diff
@@ -79,6 +79,11 @@ export default class MusicBlocklyWorkspace {
       }
     });
 
+    // Remove two default entries in the toolbox's Functions category that
+    // we don't want.
+    delete Blockly.Blocks.procedures_defreturn;
+    delete Blockly.Blocks.procedures_ifreturn;
```

Comment:
Yes, I think this is the least bad approach, especially because we'd rather not reimplement their code in our own repo, and risk it going out of date.  Going forward, it would be great if Blockly offered a way to request which elements are included in these built-in dynamic categories.